benchadapt/python/benchadapt/adapters/asvbench.py

```diff
@@ -0,0 +1,179 @@
+import itertools
+import json
+import os
+from datetime import datetime
+from pathlib import Path
+from typing import Any, Dict, List
+
+import numpy as np
```

Comment:
Would you mind adding `numpy` to `requirements.txt`?

Reply:
Thanks for the review!